.github/workflows/ci.yaml

```diff
@@ -5,13 +5,18 @@ on:
   push:
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  Build:
+    strategy:
+      matrix:
+        go-version: [stable, oldstable]
```

Comment:
Should this run go1.20 too?

Reply:
This seems like an easier way to manage the versions. I think it's fine if 1.20 only runs on circleci. Maybe it doesn't even need to run there.